src/main/java/com/google/devtools/build/lib/starlarkbuildapi/StarlarkRuleContextApi.java

```diff
@@ -568,7 +568,7 @@ String expandLocation(String input, Sequence<?> targets, StarlarkThread thread)
         @Param(
             name = "transitive_files",
             allowedTypes = {
-              @ParamType(type = Depset.class, generic1 = FileApi.class),
```

Comment:
I had resisted making this change in the past because "depset of file", while not an accurate description of any type in the Java program, is exactly what the user wants to read in the generated documentation---and generic1 exists only for documentation.

I agree it's weird, but I'm not sure it's bad.